pkg/fixtures/fixtures.go

```diff
@@ -55,19 +52,24 @@ type Fixture struct {
 	Fs            afero.Fs
 	APIKey        string
 	StripeAccount string
+	Skip          []string
+	Overrides     map[string]interface{}
+	Additions     map[string]interface{}
+	Removals      map[string]interface{}
 	BaseURL       string
-	responses     map[string]*gojsonq.JSONQ
+	responses     map[string]gjson.Result
 	fixture       fixtureFile
 }
 
 // NewFixture creates a to later run steps for populating test data
-func NewFixture(fs afero.Fs, apiKey, stripeAccount, baseURL, file string) (*Fixture, error) {
+func NewFixture(fs afero.Fs, apiKey string, stripeAccount string, baseURL string, file string, skip, override, add, remove []string) (*Fixture, error) {
```

Comment:
Just to be consistent, I would not repeat the `string` type since you are not repeating the `[]string` type.

```go
func NewFixture(fs afero.Fs, apiKey, stripeAccount, baseURL, file string, skip, override, add, remove []string) (*Fixture, error) {
```

Reply:
of course. will do that in the next commit